tools/bin/hawqregister

```diff
@@ -855,6 +888,79 @@ class HawqRegister(object):
         logger.info('Hawq Register Succeed.')
 
 
+class HawqRegisterPartition(HawqRegister):
```

Comment:
Where is the `__init__` function of this class? Use base class 's constructor? I am confused here.